testCourse/serverFilesCourse/python_autograder/pltest.py

```diff
@@ -91,7 +91,7 @@ def getResults(self):
 
         with open("results.json", mode="w") as out:
             json.dump(grading_result, out)
-    except:
+    except:  # noqa: E722
```

Comment:
Maybe just use an Exception here? This looks like it doesn't need to catch system calls.

Might also be worth assembling some of the dicts in this file with one liners.

Reply:
I'm actually wondering if this file is even needed - AFAICT it's not referenced or used at all within the `testCourse`?

Reply:
This might have been used as an example custom autograder? Not sure if that's necessary though.

Reply:
Maybe, though that would belong in `exampleCourse` if that were true. Let me open another PR to remove that and see if the tests still pass 🤷

Reply:
#7010